src/app/sidebar/sidebar.less

```diff
@@ -320,4 +320,13 @@
             top: -3px;
         }
     }
+
+    .updateBanner {
+        font-weight: bold;
+
+        .icon {
+            font-weight: normal;
+            font-size: 16px;
```

Comment:
this should be 15px (Martian Mono font is optimized for 12.5px and 15px -- 2.5px steps)

Reply:
This is set here so that the size of the fontawesome icon matches that of the custom SVGs in the other sidebar items. This does not change the text font sizing